doc/nrf/protocols/matter/end_product/watchdog.rst

```diff
@@ -69,27 +98,38 @@ Feeding automatically
 
         void FeedFromApp(Nrf::Watchdog::WatchdogSource *source)
         {
-	        if (source) {
-		    Nrf::PostTask([source] { source->Feed(); });
+	        if (source)
+            {
+		        Nrf::PostTask([source] { source->Feed(); });
 	        }
         }
 
-  2. Create a Matter watchdog source and provide the ``uint32_t feedingInterval``, and ``FeedingCallback callback`` optional arguments to the Matter watchdog source constructor.
+  3. Create a Matter watchdog source and provide the ``uint32_t feedingInterval``, and ``FeedingCallback callback`` optional arguments to the Matter watchdog source constructor.
 
      For example, here is a Matter watchdog source object configured to call the previously defined callback every 200 ms:
 
      .. code-block:: c++
 
         Nrf::Watchdog::WatchdogSource myWatchdog(200, FeedFromApp);
 
-  3. Enable the Global watchdog module:
+  4. Try to install the Matter watchdog source:
```

Comment:
This step should have something after the sample function about what the user should do if the attempt fails.

Reply:
I will add a note for that at the bottom of this point to cover both appoaches.